iocore/net/P_SSLNetVConnection.h

```diff
@@ -260,6 +260,17 @@ class SSLNetVConnection : public UnixNetVConnection
   /// Set by asynchronous hooks to request a specific operation.
   SslVConnOp hookOpRequested;
 
+  bool
+  getClientVerifyEnable()
+  {
+    return clientVerifyEnable;
+  }
+  void
+  setClientVerifyEnable(int enable)
```

Comment:
This cherry-pick just enables and disables client verification mode. The other patch I have allows permissive verification mode and changes all these accordingly. I will submit that patch later.